packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -252,6 +276,42 @@ export const prepareCosmosOrchestrationAccountKit = (
           return harden(coins.map(toDenomAmount));
         },
       },
+      transferWatcher: {
+        /**
+         * @param {[
+         *   { transferChannel: IBCConnectionInfo['transferChannel'] },
+         *   bigint,
+         * ]} results
+         * @param {{
+         *   destination: ChainAddress;
+         *   opts: IBCMsgTransferOptions;
+         *   token: Coin;
+         * }} ctx
+         */
+        onFulfilled(
+          [{ transferChannel }, timeoutTimestamp],
+          { opts, token, destination },
+        ) {
+          const results = E(this.facets.helper.owned()).executeEncodedTx([
+            Any.toJSON(
+              MsgTransfer.toProtoMsg({
+                sourcePort: transferChannel.portId,
+                sourceChannel: transferChannel.channelId,
+                token,
+                sender: this.state.chainAddress.value,
+                receiver: destination.value,
+                timeoutHeight: opts?.timeoutHeight ?? {
+                  revisionHeight: 0n,
+                  revisionNumber: 0n,
+                },
+                timeoutTimestamp,
+                memo: opts?.memo ?? '',
```

Comment:
I see `memo` is optional on `IBCMsgTransferOptions` but the fn signature says `opts` must be defined.
```suggestion
                memo: opts.memo ?? '',
```
or make `opts` optional

Reply:
Good catch, thanks